spec/03-plugins/24-response-rate-limiting/05-integration_spec.lua

```diff
@@ -21,28 +22,6 @@ local REDIS_PASSWORD = "secret"
 
 local SLEEP_TIME = 1
 
-local function redis_connect()
-  local red = redis:new()
-  red:set_timeout(2000)
-  assert(red:connect(REDIS_HOST, REDIS_PORT))
-  local red_version = string.match(red:info(), 'redis_version:([%g]+)\r\n')
-  return red, assert(version(red_version))
-end
-
-local function flush_redis(red, db)
-  assert(red:select(db))
-  red:flushall()
-end
-
-local function add_redis_user(red)
-  assert(red:acl("setuser", REDIS_USER_VALID, "on", "allkeys", "+incrby", "+select", "+info", "+expire", "+get", "+exists", ">" .. REDIS_PASSWORD))
```

Comment:
We've replaced that with `redis_helper.add_admin_user` that has more privileges but I don't think it impacts the tests.